dts/bindings/serial/nuvoton,numaker-uart.yaml

```diff
@@ -0,0 +1,24 @@
+# (c) Nuvoton Technology Corp. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, Numaker-UART
+
+compatible: "nuvoton,numaker-uart"
+
+include: [uart-controller.yaml, reset-device.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  resets:
+    required: true
+
+  clocks:
+    required: true
+
+  pinctrl-0:
```

Comment:
require pinctrl-names as well?

Reply:
Thanks of this review.
Above issues are fixed and updated.

Reply:
Hi @gmarull,
Fixed and please have a look into the update.